File: primes.py
import sys

def primes (max):
	array = []
	for i in range(2, (max + 1)):
		isPrime = True
		for n in range ((i - 1), 2, -1):
			if (i % n == 0):
				isPrime = False
				break
		if (isPrime): array.append(i)
	return array

print(primes(int(sys.argv[1])))

File: primes.cs
List<int> primes (int max) {
	List<int> array = new List<int>();
	for (int i = 2; i <= max; i++) {
		bool isPrime = true;
		for (int n = (i - 1); n >= 2; n--) {
			if (i % n == 0) {
				isPrime = false;
				break;
			}
		}
		if (isPrime) array.Add(i);
	}
	return array;
}

Console.WriteLine(string.Join(',', primes(Int32.Parse(args[0]))));
